lms/djangoapps/instructor/views/api.py

```diff
@@ -28,6 +28,7 @@
 from django.urls import reverse
 from django.core.validators import validate_email
 from django.db import IntegrityError, transaction
+from django.db import models
```

Comment:
It looks redundant. We can import models on line 31.

Reply:
Ooo, I miss it, thanks.